Tests/Realm.Tests/Sync/SessionTests.cs

```diff
@@ -750,18 +756,33 @@ public void Session_OnSessionError()
             });
         }
 
-        [TestCase(ProgressMode.ForCurrentlyOutstandingWork)]
-        [TestCase(ProgressMode.ReportIndefinitely)]
-        public void SessionIntegrationTest_ProgressObservable(ProgressMode mode)
+#pragma warning disable CS0618 // Type or member is obsolete
+        [Test]
+        public void SessionIntegrationTest_ProgressObservable(
```

Comment:
Previously this test was only testing for callbacks being invoked because the notifications were unreliable. We should improve the test to now validate that meaningful progress is being reported and not that it was just invoked.